build/Targets/Tools.props

```diff
@@ -20,7 +20,7 @@
         - Update links and version numbers in docs/contributing/Building%2C%20Debugging%2C%20and%20Testing%20on%20Windows.md
         - Email the team to ask them to upgrade to the new SDK
     -->
-    <dotnetSdkVersion>2.1.300-rtm-008866</dotnetSdkVersion>
+    <dotnetSdkVersion>2.1.400</dotnetSdkVersion>
```

Comment:
Consider using 2.1.401 instead. It's the latest patch. We should just use that if we're making the change here.

Reply:
The source build is currently using a preview build of .401 (https://github.com/dotnet/source-build/blob/master/DotnetCLIVersion.txt). Would I need to specify the same preview build or would the release build work here?

Reply:
Thanks for thinking to check that. I've verified other repositories have moved over to RTM so we should be fine. In general for us moving ahead a patch version is just fine.

Reply:
I've moved this change to a separate PR (https://github.com/dotnet/roslyn/pull/29722)